docs/sql-reference/sql-statements/data-definition/CREATE TABLE.md

```diff
@@ -376,34 +376,78 @@ Since version 3.0, the primary key and sort key are decoupled in the Primary Key
+    | "storage_cooldown_time" = "yyyy-MM-dd HH:mm:ss" }
 )
 ```
 
-**storage_medium**: the initial storage medium, which can be set to SSD or HDD.
+- `storage_medium`: the initial storage medium, which can be set to `SSD` or `HDD`. Make sure that the type of storage medium you explicitly specified is consistent with the BE disk types for your StarRocks cluster specified in the BE static parameter `storage_root_path`.<br>
 
-> **NOTE**
->
-> - From 2.5.1, the system automatically infers storage medium based on BE disk type if `storage_medium` is not explicitly specified. Inference mechanism: If `storage_root_path` reported by BEs contain only SSD, the system automatically sets this parameter to SSD. If `storage_root_path` reported by BEs contain only HDD, the system automatically sets this parameter to HDD. If `storage_root_path` reported by BEs contain both SSD and HDD, the system automatically sets this parameter to SSD. From 2.3.10, 2.4.5, 2.5.4 onwards, if `storage_root_path` reported by BEs contain both SSD and HDD and the property `storage_cooldown_time` is specified, `storage_medium` is set to SSD; if the property `storage_cooldown_time` is not specified, `storage_medium` is set to HDD.
-> - If the FE configuration item `enable_strict_storage_medium_check` is set to `true`, the system strictly checks BE disk type when you create a table. If the storage medium you specified in CREATE TABLE is inconsistent with BE disk type, an error "Failed to find enough host in all backends with storage medium is SSD|HDD." is returned and table creation fails. If `enable_strict_storage_medium_check` is set to `false`, the system ignores this error and forcibly creates the table. However, cluster disk space may be unevenly distributed after data is loaded.
+    If the FE configuration item `enable_strict_storage_medium_check` is set to `true`, the system strictly checks BE disk type when you create a table. If the storage medium you specified in CREATE TABLE is inconsistent with BE disk type, an error "Failed to find enough host in all backends with storage medium is SSD|HDD." is returned and table creation fails. If `enable_strict_storage_medium_check` is set to `false`, the system ignores this error and forcibly creates the table. However, cluster disk space may be unevenly distributed after data is loaded.<br>
+
+    From 2.3.6, 2.4.2, 2.5.1, and 3.0 onwards, the system automatically infers storage medium based on BE disk type if `storage_medium` is not explicitly specified.<br>
+
+  - The system automatically sets this parameter to SSD in the following scenarios:
+
+    - The disk types reported by BEs (`storage_root_path`) contain only SSD.
+    - The disk types reported by BEs (`storage_root_path`) contain both SSD and HDD. Note that from 2.3.10, 2.4.5, 2.5.4, and 3.0 onwards, the system sets `storage_medium` to SSD when `storage_root_path` reported by BEs contain both SSD and HDD and the property `storage_cooldown_time` is specified.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/dc666f9a-258d-4d45-b20b-b424d5034fe0)
See the comment on Line 399.